김신아/index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="calculator.css" />
+  </head>
+  <body>
+    <div class="entire_calculator">
+      <div class="result_display">
+        <div class="result_wrapper">
+          <div class="result_processing"></div>
+          <div class="result"></div>
+        </div>
+      </div>
+      <div>Ac</div>
+      <div>7</div>
+      <div>8</div>
+      <div>9</div>
+      <div>x</div>
+      <div>4</div>
+      <div>5</div>
+      <div>6</div>
+      <div>-</div>
+      <div>1</div>
+      <div>2</div>
+      <div>3</div>
+      <div>+</div>
+      <div>0</div>
+      <div>=</div>
+    </div>
+    <script>
+      let result = 0;
+      let resultProcessing = 0;
+      const numberList = document.body.querySelectorAll(
+        ".entire_calculator div"
+      );
+
+      numberList[4].addEventListener("click", clear);
+      numberList[5].addEventListener("click", function () {
+        numberRecall("7");
+      });
```

Comment:
```suggestion
      numberList[5].addEventListener("click", function (event) {
        numberRecall(event.innerHTML);
      });
```

이렇게 값을 직접 할당하는 것이 아닌, DOM 요소에서 렌더링되고 있는 값을 이용할 수 있어요!